doc/source/conf.py

```diff
@@ -297,6 +297,18 @@
     'pd.options.display.encoding="utf8"'
     ]
 
+# monkeypatch in accessor methods so we can refer to them without creating
+# an actual Series object.
+from pandas.core.strings import StringMethods
+pandas.Series.str = StringMethods
+
+from pandas.tseries.common import DatetimeProperties, TimedeltaProperties
+class CombinedDatetimelikeProperties(DatetimeProperties, TimedeltaProperties):
+    pass
+pandas.Series.dt = CombinedDatetimelikeProperties
+
+pandas.Series.cat = Categorical
```

Comment:
Is this one already used? I think not yet, but maybe we should? (although here it is less clear, as how it is now in the api.rst is (like `Categorical.categories`) is a valid object method in itself)